### ISSUE TITLE
compatible issues spree 2.4

### ISSUE THREAD

**Issue by techverx** [stefansenk/spree-blogging-spree#15]
I just want to ask that, is this gem is compatible with spree 2-4-stable branch because i am facing lot of issues with spree-blogging-spree master branch.

Thanks

**Comment by forever-sumit** [comment]
@techverx 
yes, right now this gem not support with spree 2-4-stable. and non of us is working on upgrading.
you are welcome for the pull request if you fix the issues and upgrade for spree-2-4-stable
